e2e/scripts/setup-jira-playwright.ts

```
@@ -0,0 +1,300 @@
+    return exec(cmd);
+  } catch {
+    return '';
+  }
+}
+
+/**
+ * Generate license using atlassian-agent.jar
+ */
+function generateLicense(serverId: string): string {
+  const cmd = `docker exec ${CONTAINER_NAME} java -jar /var/agent/atlassian-agent.jar -d -p jira -m test@example.com -n "Test" -o "Test Org" -s "${serverId}"`;
+  const output = exec(cmd, 60000);
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)